types/index.d.ts

```diff
@@ -34,10 +34,38 @@ export interface RenderOptions<
   Q extends Queries = typeof queries,
   Container extends Element | DocumentFragment = HTMLElement
 > {
+  /** By default, React Testing Library will create a div and append that div to the document.body and this is 
+   *  where your React component will be rendered. If you provide your own HTMLElement container via this option,
+   *  it will not be appended to the document.body automatically.
+   *
+   *  For example: If you are unit testing a tablebody element, it cannot be a child of a div. In this case, you can
+   *  specify a table as the render container.
+   * 
+   *  [Container](https://testing-library.com/docs/react-testing-library/api/#container)
+   */
   container?: Container
+  /** If the container is specified, then this defaults to that, otherwise this defaults to document.body. This is used as 
+   *  the base element for the queries as well as what is printed when you use debug().
+   *  
+   *  [BaseElement](https://testing-library.com/docs/react-testing-library/api/#baseelement)
```

Comment:
Not a fan of these non-descriptive links. What do you think about just 
```suggestion
   * @see https://testing-library.com/docs/react-testing-library/api/#baseelement
```

Reply:
Hmm what do you think about this `@links` ? The think with `@see` is that you loose the hyperlink on the hint :( @eps1lon

Reply:
> The think with @see is that you loose the hyperlink on the hint :(

Will check. I was under the impression VSCode did make these clickable.